intro-HPC/ch_running_jobs_with_input_output_data.tex

```diff
@@ -751,10 +752,35 @@ \subsection{Setting per-member VO quota}
         100\% can actually be used.
 \end{enumerate}
 
-\begin{figure}[h]
+\begin{figure}[!htbp]
   \caption{Setting per-member quota.}
   \centering
     \includegraphics[width=\textwidth]{ch6-VO-per-member-storage}
-\end{figure}
+\end{figure}\label{fig:VO-per-member-storage}
+
+\subsection{VO directories}
+\label{subsec:vo-directories}
+
+When you're a member of a VO, there will be some additional filesystems available:
+
+\begin{description}
+    \item[VO scratch (\texttt{\$VSC\_SCRATCH\_VO})]
+        A directory on the shared scratch filesystem shared by the members of your
+        VO, where additional storage quota can be provided (see \autoref{subsec:requesting-more-storage-space}).
+        You can use this as an alternative to your personal \verb|$VSC_SCRATCH|
+        directory (see \autoref{subsec:scratch-directory}).
+
+    \item[VO data (\texttt{\$VSC\_DATA\_VO})]
+        A directory on the shared data filesystem shared by the members of your
+        VO, where additional storage quota can be provided (see \autoref{subsec:requesting-more-storage-space}).
+        You can use this as an alternative to your personal \verb|$VSC_DATA|
+        directory (see \autoref{subsec:data-directory}).
+
+\end{description}
+
+If you put \verb|_USER| after each of these variable names, you can see your
+personal folder in these filesystems. For example: \verb|$VSC_DATA_VO_USER|
+is your personal folder in your VO data filesystem (this is equivalent to
+\verb|$VSC_DATA_VO/$USER|).
```

Comment:
Also mention `$VSC_SCRATCH_VO_USER`: `, and analogous for \verb|$VSC_SCRATCH_VO_USER|.`